Upstream-Status: Inappropriate [embedded specific]

By default gcc places 64 bit libs in a lib64 directory. This makes it use 
"lib" instead.

RP 25/7/10

Index: git/gcc/config/i386/t-linux64
===================================================================
--- git.orig/gcc/config/i386/t-linux64
+++ git/gcc/config/i386/t-linux64
@@ -24,8 +24,8 @@
 # MULTILIB_OSDIRNAMES according to what is found on the target.
 
 MULTILIB_OPTIONS = m64/m32
-MULTILIB_DIRNAMES = 64 32 
-MULTILIB_OSDIRNAMES = ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)
+MULTILIB_DIRNAMES = . 32
+MULTILIB_OSDIRNAMES = . $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)
 
 LIBGCC = stmp-multilib
 INSTALL_LIBGCC = install-multilib
Index: git/gcc/config/i386/linux64.h
===================================================================
--- git.orig/gcc/config/i386/linux64.h
+++ git/gcc/config/i386/linux64.h
@@ -25,5 +25,5 @@ see the files COPYING3 and COPYING.RUNTI
 <http://www.gnu.org/licenses/>.  */
 
 #define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
-#define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux-x86-64.so.2"
-#define GLIBC_DYNAMIC_LINKERX32 "/libx32/ld-linux-x32.so.2"
+#define GLIBC_DYNAMIC_LINKER64 "/lib/ld-linux-x86-64.so.2"
+#define GLIBC_DYNAMIC_LINKERX32 "/lib/ld-linux-x32.so.2"
Index: git/gcc/config/i386/t-linux-x32
===================================================================
--- git.orig/gcc/config/i386/t-linux-x32
+++ git/gcc/config/i386/t-linux-x32
@@ -31,8 +31,8 @@
 # 	/libx32 has x32 libraries.
 #
 MULTILIB_OPTIONS = m64/m32/mx32
-MULTILIB_DIRNAMES = 64 32 x32
-MULTILIB_OSDIRNAMES = ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib) ../libx32
+MULTILIB_DIRNAMES = 64 32 .
+MULTILIB_OSDIRNAMES = ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib) .
 
 LIBGCC = stmp-multilib
 INSTALL_LIBGCC = install-multilib
